js/bybit.js

```diff
@@ -3203,6 +4184,101 @@ module.exports = class bybit extends Exchange {
         return this.parseOrder (order);
     }
 
+    async createUnifiedMarginOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        if (!market['linear'] && !market['option']) {
+            throw new NotSupported (this.id + ' createOrder does not allow inverse market orders for ' + symbol + ' markets');
```

Comment:
first isn't easier to check `if market['inverse']` ? 🙂 
Second, I think this message is a bit confusing, if you are in the unifiedMargin mode you still can place orders for inverse markets the only difference is that will use the normal "margin", so should we call the normal "create order" method if the user chooses an inverse market?

Reply:
@sc0Vu yeah it can be a bit confusing, but if we want to throw here i think we should improve the error message guiding the user to the right path